registration-processor/registration-processor-message-sender-impl/src/main/java/io/mosip/registration/processor/message/sender/service/impl/MessageNotificationServiceImpl.java

```diff
@@ -19,6 +23,7 @@
 import org.apache.commons.io.IOUtils;
 import org.apache.commons.lang.StringUtils;
 import org.apache.commons.lang.exception.ExceptionUtils;
+import org.h2.util.New;
```

Comment:
might have been added by mistake, make sure we dont use any wrong util by mistake.

Reply:
I have removed it